library/std/intrinsic.qs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Quantum.Intrinsic {
 
     /// # Summary
     /// Applies the AND gate that is more efficient for use with decomposition of multi-controlled operations.
+    /// Please note that target qubit must be in $\ket{0}$ state.
```

Comment:
I believe we need to avoid markdown in `Summary` and `Input` sections for compatibility with VS Code. You can use unicode here instead, and don't need the "Please" at the front:
```suggestion
    /// Note that target qubit must be in |0⟩ state.
```